WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -478,7 +478,7 @@ protected function mergeFunctionLists() {
 				);
 			}
 
-			$this->escapingFunctions = $this->merge_custom_array(
+			$this->escapingFunctions                = $this->merge_custom_array(
```

Comment:
As noted in https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1122#issuecomment-328516737, I'd prefer to see a blank line added after this assignment rather than realigning it with the assignments below it.

Reply:
I've done this now for the large alignments (in the mergeFunctionLists/Whitelist functions) and in one or two other places where it made sense.

Added the change as a separate commit. If ok, I'll squash it into the previous one.
If you'd like more changes, let me know, though from reviewing the others, these groupings make sense (to me).

Reply:
Thanks, looks good to me.

Reply:
Ok, squashed the wo code base fix related commits into one. Ready to merge again.